tests/test_integrations.py

```diff
@@ -61,7 +61,7 @@ def run_xonsh(cmd, stdin=sp.PIPE, stdout=sp.PIPE, stderr=sp.STDOUT, single_comma
     env["XONSH_SHOW_TRACEBACK"] = "1"
     env["RAISE_SUBPROC_ERROR"] = "0"
     env["PROMPT"] = ""
-    xonsh = "xonsh.bat" if ON_WINDOWS else "xon.sh"
+    xonsh = "xonsh.exe" if ON_WINDOWS else "xonsh"
```

Comment:
I think you could just have "xonsh" in both cases. It will get the "exe" version on windows

Reply:
Yes, that's right.  Added to latest push, along with simplifying PATH for integration tests that run xonsh in subproc. Did you know that path to site-packages in venv in Windows does not have 'pythonxx/' where Linux does?  I don't think xonsh should try to track things in such detail, we'll forever be a step behind.